packages/web/src/components/EmailConfirmMessage/EmailConfirmMessage.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { useEffect } from 'react';
-import { useLocation } from 'react-router-dom';
 import { AlertPlaceholder, ErrorAlert, InfoAlert } from 'elements/alerts';
-import { QueryParams } from '../../constants/query-params';
+import { useSelector } from 'react-redux';
+
+import { getQuery } from 'store/router/selectors';
```

Comment:
If you have eslint configured this import sequence shouldn't have passed

Reply:
OK, we don't have this rule configured yet. But generally please keep imports together, and keep absolute imports higher, like so:

```ts
import React, { useEffect } from 'react';
import { useSelector } from 'react-redux';
import { AlertPlaceholder, ErrorAlert, InfoAlert } from 'elements/alerts';
import { getQuery } from 'store/router/selectors';
```